release-notes.md

```diff
@@ -1,4 +1,5 @@
 ## NEXT
+* Bugfix 132060 : the opening date will be populated with the ProposedAcadamenyOpeningDate value
```

Comment:
Could you follow convention with the naming here, this is also incorrect as they opening date will not be populated as you have removed it! haha